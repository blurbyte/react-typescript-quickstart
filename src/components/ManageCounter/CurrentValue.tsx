import styled from '@styles/styledComponents';
import { Color } from '@styles/colors';

const CurrentValue = styled.p`
  display: inline-flex;
  justify-content: center;
  align-items: center;
  line-height: 1;
  height: 3.6rem;
  font-size: 2.4rem;
  font-weight: 700;
  color: ${Color.White};
  background: ${Color.Blue};
  padding: 0 1.4rem;
  border-radius: 6px;
  min-width: 8rem;
  margin: 1rem 0;
`;

export default CurrentValue;
