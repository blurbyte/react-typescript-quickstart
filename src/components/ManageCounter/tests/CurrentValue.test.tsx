import * as React from 'react';
import { shallow } from 'enzyme';

import CurrentValue from '../CurrentValue';

describe('CurrentValue', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<CurrentValue />);
    expect(wrapper).toMatchSnapshot();
  });
});
