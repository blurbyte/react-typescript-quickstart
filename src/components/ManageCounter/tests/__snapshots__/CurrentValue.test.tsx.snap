// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentValue renders correctly 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  line-height: 1;
  height: 3.6rem;
  font-size: 2.4rem;
  font-weight: 700;
  color: #ffffff;
  background: #4a6cbb;
  padding: 0 1.4rem;
  border-radius: 6px;
  min-width: 8rem;
  margin: 1rem 0;
}

<p
  className="c0"
/>
`;
